.cirrus.yml

```diff
@@ -386,3 +386,24 @@ task:
     - cp src/qt/android/build/outputs/apk/debug/android-debug.apk ${CIRRUS_WORKING_DIR}/unsecure_android.apk
   unsecure_android_apk_artifacts:
     path: "unsecure_android.apk"
+
+task:
+  name: 'ARM32 Android APK [jammy]'
+  alias: android32
+  << : *CONTAINER_DEPENDS_TEMPLATE
+  container:
+    image: ubuntu:jammy
+  android_sdk_cache:
+    folder: "depends/SDKs/android"
+    fingerprint_key: "ANDROID_API_LEVEL=28 ANDROID_BUILD_TOOLS_VERSION=28.0.3 ANDROID_NDK_VERSION=23.2.8568313"
+  depends_sources_cache:
+    folder: "depends/sources"
+    fingerprint_script: git rev-parse HEAD:depends/packages
+  << : *MAIN_TEMPLATE
+  env:
+    << : *CIRRUS_EPHEMERAL_WORKER_TEMPLATE_ENV
+    FILE_ENV: "./ci/test/00_setup_env_android32.sh"
+  copy_artifacts_script:
+    - cp src/qt/android/build/outputs/apk/debug/android-debug.apk ${CIRRUS_WORKING_DIR}/unsecure_android_32bit.apk
+  unsecure_android_apk_32bit_artifacts:
```

Comment:
@hebasto will this change the url for the .zip to unsecure_android_apk_32bit.zip?

Reply:
> @hebasto will this change the url for the .zip to unsecure_android_apk_32bit.zip?

It does. See https://cirrus-ci.org/guide/writing-tasks/#latest-build-artifacts.

We use the following URL format:
```
https://api.cirrus-ci.com/v1/artifact/github/<USER OR ORGANIZATION>/<REPOSITORY>/<TASK NAME OR ALIAS>/<ARTIFACTS_NAME>.zip
```